lib/OpenQA/Utils.pm

```diff
@@ -507,7 +507,7 @@ sub asset_type_from_setting {
     if ($setting eq 'ISO' || $setting =~ /^ISO_\d+$/) {
         return 'iso';
     }
-    if ($setting =~ /^HDD_\d+$/) {
+    if ($setting =~ /^HDD_\d+$/ || $setting =~ /^UEFI_PFLASH_\w+$/) {
```

Comment:
I guess it would help if you add a comment here describing why UEFI_PFLASH is also applicable.

Reply:
If it needs explaining then I am not sure it should go there. My second choice would be to put it in a folder called firmware, but I'm not sure what consequences that would have.

Reply:
Well, I would of course also prefer to have no comment when the code is self-explanatory

Reply:
To me it is